otel-worker-core/src/data/models.rs

```diff
@@ -70,6 +70,11 @@ impl From<api::models::Span> for Span {
     }
 }
 
+#[derive(Deserialize)]
+pub struct QueryParams {
```

Comment:
Can you prefix this with `TracesList` as we might have other QueryString parameters.